src/index.test.js

```diff
@@ -324,3 +324,59 @@ describe('time zone is carried over after a business-day operation', () => {
     expect(nyPlusTwo.zoneName).toBe('America/New_York');
   });
 });
+
+describe('business day diff', () => {
+  it('knows two identical DateTimes have a business day diff of 0', () => {
+    const targetDate = DateTime.local()
+      .startOf('day')
+      .plus({ hours: 2 });
+
+    expect(DateTime.local().businessDiff(targetDate)).toEqual(0);
+  });
+
+  it('knows there are 3 business days between two dates that are 3 business days apart', () => {
+    const myCompanyTakesNoHolidays = [];
+    const startDate = DateTime.local().startOf('day');
```

Comment:
is `.startOf` necessary for the tests?

Reply:
Removing them on local seems to not affect the tests, perhaps we can refactor them out.